auto/update_snap_counts.R

```diff
@@ -2,9 +2,11 @@ pkgload::load_all()
 
 scrape_snaps <- function(){
 
-  completed_games <- piggyback::pb_download_url(file = "scraped_games.csv",
-                                                repo = "nflverse/nflverse-pfr",
-                                                tag = "snap_counts_raw") |>
+  completed_games <- piggyback::pb_download_url(
```

Comment:
csv_from_url would be fine, I was lazy at the time

Reply:
done